### ISSUE TITLE
wkhtmltopdf footer and header not rendered properly if total pages over 70+

### ISSUE THREAD

**Issue by cyyeap** [wkhtmltopdf/wkhtmltopdf#4079]
C:\Program Files\wkhtmltopdf\V0_12_4\bin\wkhtmltopdf.exe - arguments: --cache-dir C:\Temp\wkt\folder\ --javascript-delay 8000 --window-status ready_to_print  --dpi 300 --margin-bottom 0 --margin-left 0 --margin-right 0 --margin-top 0 --header-spacing 3 -T 33mm --header-html "[https:// angular header"footer-spacing 3 -B 25mm --footer-html "http:// angular footer url" --orientation Landscape "http:// body url" C:\Temp\blah.pdf 

Header and footer are called via https, but when there're over 70+ they cannot be rendered. 
With different errors/warnings:
* Warning: Received createRequest signal on a disposed ResourceObject's NetworkAccessManager. This might be an indication of an iframe taking too long to load.
* Warning: SSL error ignored                                         
* Warning: Received createRequest signal on a disposed ResourceObject's NetworkAccessManager. This might be an indication of an iframe taking too long to load.
* Exit with code 1 due to network error: ProtocolUnknownError

Does anyone know how to resolve this? 

Thanks

**Comment by Tomsgu** [comment]
Hi @cyyeap,

can you try it in 0.12.5 version?

**Comment by cyyeap** [comment]
Yes, we've tried. Each page calls two requests just to retrieve header and footer, if more than N pages, it's way too many requests. We're planning to move header/footer to MVC.  It'd be wonderful that wkhtmltopdf doesn't to that many calls repetitively, potentially cache header/footer without calling it so many times.
